Adapter/GD.php

```diff
@@ -213,6 +213,56 @@ public function sepia()
     /**
      * {@inheritdoc}
```

Comment:
`gaussianBlur` does not exists on parent class.

This should be added on interface or have is proper php doc.

@Gregwar Adding to interface would be BC. A new major version should be released with an upgrade note.